### DISCUSSION TITLE
Die Ziffer 6 hat einen Spiegel

### DISCUSSION THREAD

**Discussion by nektar86** [jomjol/AI-on-the-edge-device#2052] [General]
Hallo.
Die Ziffer 6 der letzten Walze meines Gaszählers hat einen Spiegel.
So das diese Falsch erkannt wird. Kann diese Ziffer auch eingelernt werden?
Ich habs jetzt soweit am laufen das in den letzten 8 Std. nur ein Fehler aufgetreten ist.
![9 6_main_dig8_20230219-203326](https://user-images.githubusercontent.com/97253478/219973042-092dcc3c-e2e4-42cb-97b2-a6a010465105.jpg)

Gruß Christian

**Comment by caco3** [comment]
ja, ich denke, du könntest das einlernen. 
Aber die frage ist, ob du dieses digit überhaupt brauchst.
Ich habe es bei mir weggelassen, da der zähler sowieso schnell genug zählt.

**Comment by nektar86** [comment]
Was meinst du mit das der Zähler sowieso schnell genug zählt?
Heute hatte ich wieder eine Fehlermeldung wegen dem Spiegel in der 6.
Und eine wegen der Blendung der 6. Ziffer. Dort wird die 7 manchmal als 2 gesehen.

**Reply by caco3** [reply]
Ich habe bei meinem Gaszähler die 3. Stelle nach dem Koma nicht verwendet, da sich pro Interval bereits die 2. Stelle genügend ändert.

**Comment by nektar86** [comment]
Achse ok. Das müsste ich mal überprüfen.
Bei der 3. Stelle muss ich mal schauen ob ich die Spiegelung noch weg bekomme.
